colearn_keras/keras_mnist.py

```diff
@@ -177,13 +228,15 @@ def prepare_learner(data_loaders: Tuple[PrefetchDataset, PrefetchDataset, Prefet
         minimise_criterion=False,
         model_fit_kwargs={"steps_per_epoch": steps_per_epoch},
         model_evaluate_kwargs={"steps": vote_batches},
+        diff_priv_config=diff_priv_config,
```

Comment:
This comment is for the loader arguments:

We do we use only data_loaders[0] and [1] twice, when we have 0-1-2 as per L154?

Reply:
hmm this might be similar at other places too: we declare train/vote/test datasets, but end up voting on the test dataset and not using the vote dataset, see https://github.com/fetchai/colearn/blame/master/colearn_examples/ml_interface/keras_mnist_diffpriv.py at L60-62 and L128-131